NEWS

```diff
@@ -8,6 +8,8 @@ Release 0.9.1 on 2020-12-28
 ---------------------------
 
   * Client minimization (controlled by the attribute 'minimized' of every client)
+  * Client window titles (controlled by the theme attributes 'title_height',
```

Comment:
Should probably open a new "Current git version" section for this

Reply:
Right, Thanks! I've fixed it.

Reply:
Thank you!